### ISSUE TITLE
Remove notification when marked as read

### ISSUE THREAD

**Issue by peterpeerdeman** [part-up/app#40]
as an app user, I want notifications that are pushed to my mobile phone to be revoked/removed when I mark notifications as read on the web interface.

**Comment by ralphboeije** [comment]
And the other way around: when you look at an update detail in the app, I want the notification in the web app to be marked as read (which also then is visible in the part-ups tab in the app - lower notification number next to the part-up).

**Comment by jessedvrs** [comment]
@ralphboeije What you're requesting here does already work. (And has nothing to do with mobile push notifications).

**Comment by ralphboeije** [comment]
What I meant is that when you:
1. Receive a push notification on your mobile phone
2. Click to view it in the app
3. Go to the update details in the app (of this notification)
4. Click the part-ups tab
5. View the orange counter number behind the part-up from which this notification came
You see that the counter has not been decreased by viewing the update detail in the app. Which also holds true (of course) for counters next to the part-ups in the web app user menu. This is quite annoying, because you think you still missed something on the part-up, but in fact you have seen everything.

**Comment by peterpeerdeman** [comment]
let's separate these 2 feature requests, these are completely different functionalities

**Comment by ralphboeije** [comment]
just did and added #86

**Comment by eriksoon** [comment]
I would like to be able to mark the notification as read (although I didn't read it by going to the page). I would like this, because I'm often partner in a part-up just for one or two activities and do not want to welcome every new supporter or read every update on activities that do not apply to me. Does this fit this issue?

**Comment by jessedvrs** [comment]
No, this issue is about revoking push notifications sent to mobile devices (part-up app). Please create a feature request.

**Comment by eriksoon** [comment]
Ok just made issue #103
